ci/conda_env_cpp.txt

```diff
@@ -15,7 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
-aws-sdk-cpp
+# workaround for https://issues.apache.org/jira/browse/ARROW-13134
+aws-sdk-cpp<1.8
```

Comment:
cc @xhochy

Reply:
@pitrou I've pinned aws sdk in this PR. I assume the issue is still present, can we merge this?

Reply:
It should be `aws-sdk-cpp<1.9`.